app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PostsController < ApplicationController
```

Comment:
- [ ] Please add an `#index` action here in order to display data on the posts index page.